src/blogtoblog/BlogToBlogImporter.ts

```diff
@@ -24,24 +24,38 @@ export default class BlogToBlogImporter extends PageImporter {
     return fetch(url);
   }
 
+  createTable(data: (string|Element)[][], document: Document) {
+    const table = document.createElement('table');
+
+    data.forEach((row: (string|Element)[], index) => {
+      const tr = document.createElement('tr');
+
+      row.forEach((cell: string|Element) => {
+        const t = document.createElement(index === 0 ? 'th' : 'td');
+        if (typeof cell === 'string') {
+          t.innerHTML = cell;
+        } else {
+          t.append(cell);
+        }
+        tr.appendChild(t);
+      });
+      table.appendChild(tr);
+    });
+
+    return table;
+  }
+
   computeBlockName(str: string) {
-    // TODO: handle dash to spaces
     return str
-        .replace(/\s(.)/g, (s) => { return s.toUpperCase(); })
-        .replace(/\s/g, '')
-        .replace(/^(.)/, (s) => { return s.toLowerCase(); });
+      .replace(/-/g, ' ')
+      .replace(/\s(.)/g, (s) => { return s.toUpperCase(); })
+      .replace(/^(.)/g, (s) => { return s.toUpperCase(); });
```

Comment:
very cool

Reply:
well... it is probably doable with one replace less. But would be less readable.